ramls/source-record-domain-event.json

```diff
@@ -0,0 +1,97 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "description": "Source record domain event data model",
+  "javaType": "org.folio.rest.jaxrs.model.SourceRecordDomainEvent",
+  "type": "object",
+  "additionalProperties": false,
+  "properties": {
+    "id": {
+      "description": "UUID",
+      "type": "string",
+      "pattern": "^[0-9a-fA-F]{8}-[0-9a-fA-F]{4}-[1-5][0-9a-fA-F]{3}-[89abAB][0-9a-fA-F]{3}-[0-9a-fA-F]{12}$"
+    },
+    "eventType": {
+      "type": "string",
+      "enum": ["SOURCE_RECORD_CREATED", "SOURCE_RECORD_UPDATED"],
+      "description": "Source record domain event type"
+    },
+    "sourceRecordDomainEventMetadata": {
+      "description": "Event metadata",
+      "type": "object",
+      "additionalProperties": false,
+      "properties": {
+        "eventTTL": {
+          "description": "Time-to-live (TTL) for event in minutes",
+          "type": "integer"
+        },
+        "correlationId": {
+          "description": "Id to track related events, can be a meaningful string or a UUID",
+          "type": "string"
+        },
+        "originalEventId": {
+          "description": "Id of the event that started the sequence of related events",
+          "type": "string",
+          "pattern": "^[0-9a-fA-F]{8}-[0-9a-fA-F]{4}-[1-5][0-9a-fA-F]{3}-[89abAB][0-9a-fA-F]{3}-[0-9a-fA-F]{12}$"
+        },
+        "publisherCallback": {
+          "description": "Allows a publisher to provide a callback endpoint or an error Event Type to be notified that despite the fact that there are subscribers for such an event type no one has received the event within the specified period of time",
+          "type": "object",
+          "properties": {
+            "endpoint": {
+              "description": "Callback endpoint",
+              "type": "string"
+            },
+            "eventType": {
+              "description": "Error Event Type",
+              "type": "string"
+            }
+          }
+        },
+        "createdDate": {
+          "description": "Timestamp when event was created",
+          "type": "string",
+          "format": "date-time"
+        },
+        "publishedDate": {
+          "description": "Timestamp when event was initially published to the underlying topic",
+          "type": "string",
+          "format": "date-time"
+        },
+        "createdBy": {
+          "description": "Username of the user whose action caused an event",
+          "type": "string"
+        },
+        "publishedBy": {
+          "description": "Name and version of the module that published an event",
+          "type": "string"
+        }
+      },
+      "required": [
+        "eventTTL",
+        "publishedBy"
+      ]
+    },
+    "eventPayload": {
+      "type": "string",
+      "description": "The source record JSON string"
+    },
+    "tenant": {
```

Comment:
Do we need "tenant" and "ts" in the event? These fields are not mentioned in the design doc that Olamide created. https://folio-org.atlassian.net/wiki/spaces/DD/pages/62685275/Source+Record+Domain+Eventing

Reply:
These 2 fields are part of Kafka message technical design in the Folio library [folio-service-tools-spring-dev](https://github.com/folio-org/folio-service-tools/blob/master/folio-service-tools-spring-dev/src/main/java/org/folio/spring/tools/kafka/BaseKafkaMessage.java) which we use in mod-linked-data.
I wanted to follow the same here

Reply:
The file isn't needed anymore, re-used existed Event model, since it's compatible